dotnet/src/SemanticKernel.UnitTests/Functions/KernelArgumentsTests.cs

```diff
@@ -113,6 +113,6 @@ public void ItCanBeInitializedFromAnotherSettingsInstance()
         Assert.True(sut.ContainsName("fake-key"));
         Assert.Equal("fake-value", sut["fake-key"]);
 
-        Assert.Same(executionSettings, sut.ExecutionSettings);
+        Assert.Same(executionSettings, sut.ExecutionSettings?["default"]);
```

Comment:
```suggestion
        Assert.Same(executionSettings, sut.ExecutionSettings?[PromptExecutionSettings.DefaultServiceId]);
```

Reply:
Will make this change in the follow up PR